app/src/main/java/co/tinode/tindroid/AboutDialogFragment.java

```diff
@@ -26,7 +26,7 @@ public Dialog onCreateDialog(Bundle savedInstanceState) {
         }
 
         AlertDialog.Builder builder = new AlertDialog.Builder(activity);
-        View dialog = View.inflate(activity, R.layout.dialog_about, null);
+        View dialog = View.inflate(activity, R.layout.tindroid_dialog_about, null);
```

Comment:
There is no need for prefixes in the app. The prefixes should be used only in the libraries.

Reply:
That's right, I did this just to make the search/replace easier for me.
I'll remove the prefix once I have finish.